tests/robot-tests/tests/public_api/public_api_major_auto_changes.robot

```diff
@@ -0,0 +1,253 @@
+*** Settings ***
+Library             ../libs/admin_api.py
+Resource            ../libs/admin-common.robot
+Resource            ../libs/admin/manage-content-common.robot
+Resource            ../libs/public-common.robot
+Resource            ../libs/public-api-common.robot
+
+Force Tags          Admin    Local    Dev    AltersData
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+Test Setup          fail test fast if required
+Test Teardown       Run Keyword If Test Failed    record test failure
+
+
+*** Variables ***
+${PUBLICATION_NAME}=    UI tests - Public API - major auto changes %{RUN_IDENTIFIER}
+${RELEASE_1_NAME}=      Financial year 3000-01
+${RELEASE_2_NAME}=      Academic year 3010/11
+${SUBJECT_1_NAME}=      UI tests - Public API - major auto changes - subject 1 - %{RUN_IDENTIFIER}
+${SUBJECT_2_NAME}=      UI tests - Public API - major auto changes - subject 2 - %{RUN_IDENTIFIER}
```

Comment:
Good idea - we should probably start doing this in other places as well to stop namespace clashes

Reply:
Yeah it's particularly useful on pages like the Data Catalogue and Find Statistics pages and when doing any amount of `--disable-teardown` testing.